lib/client.js

```diff
@@ -35,22 +48,45 @@ class Client {
     this._callback = callback;
   }
 
+  /**
+   * Get the handle of this client.
+   * @return {RclHandle} The handle.
```

Comment:
The return type won't be listed in generated doc. Recommending `@property` syntax